### PR TITLE
Reformatting some of the code

### PR DESCRIPTION
**Pull request by Philbywhizz**
Just cleaning up some whitespace, getting rid of a few unused and shadow variables (my mistake when I rebased incorrectly).

luacheck is now down to 64 warnings :)

### CODE REVIEW

File: source/objects/lander.lua [hunk 1]
```diff
@@ -417,8 +417,6 @@ function Lander.draw()
 		love.graphics.draw(ship.image, x,y, math.rad(lander.angle), sx, sy, ox, oy)
 
 		-- draw flames
-		local ox = flame.width / 2
```
Comment (togfoxy) [inline source/objects/lander.lua:420]: Checking if these two locals are meant to be deleted?  They seem to be used 3 lines later on.

File: source/objects/lander.lua [hunk 2]
```diff
@@ -417,8 +417,6 @@ function Lander.draw()
...
-		local oy = flame.height / 2
```
Comment (togfoxy) [inline source/objects/lander.lua:421]: This local variable seems to be important.

File: source/main.lua [hunk 3]
```diff
@@ -246,11 +246,8 @@ function love.draw()
 	-- this comes BEFORE the TLfres.beginRendering
 	drawWallpaper()
 
-	local strCurrentScreen = fun.CurrentScreenName()
```
Comment (togfoxy) [inline source/main.lua:249]: Isn't this local required to draw stuff properly?

Comment (togfoxy) [reply source/main.lua:249]: I see now. strCurrentScreen has become a global in love.update.

Is that intended? Guessing you want to avoid the same call in love.draw.

File: source/objects/lander.lua (hunk 1 above)
Comment (togfoxy) [reply source/objects/lander.lua:420]: ox = ship.width / at the top on line 406 but we still need it on line 420 because this is now flame width. But 420 doesn't need to be local.

Or even better - use different names.
shipSpriteWidth
flameSpriteWidth

It's a local variable so no danger of anything going wrong.

**Review by togfoxy — CHANGES_REQUESTED** [review]

File: source/main.lua [hunk 4]
```diff
@@ -189,10 +189,10 @@ function love.load()
 		local flags = {fullscreen = false,display = 1,resizable = true, borderless = false}
 		love.window.setMode(gintScreenWidth, gintScreenHeight, flags)
     end
-	
+
 	local sock = require 'socket'	-- socket is native to LOVE but needs a REQUIRE
 	HOST_IP_ADDRESS = sock.dns.toip(sock.dns.gethostname())
-	sock = nil	
+	sock = nil
```
Comment (MadByteDE) [inline source/main.lua:195]: why settings sock = nil? I get that it's just used once but I don't the the benefit of removing it again :)

File: source/objects/lander.lua (hunk 1 above)
Comment (MadByteDE) [reply source/objects/lander.lua:420]: naming them like shipSpriteWidth would get rid of the benefits I had in mind which are for readability and shorter lines of code. Also the description is misleading since its the half of ship width. but it's used as offset for the draw function. btw the shadowing was an intentional compromise for readability. 
But if you don't like it feel free to change it to something like shipOffsetX, shipOffsetY and flameOffsetX, flameOffsetY

Comment (MadByteDE) [reply source/objects/lander.lua:420]: completely removing then without changing the function where ox and oy are used would result in a crash whenever the draw function is executed.

File: source/objects/smoke.lua [hunk 5]
```diff
@@ -66,7 +66,7 @@ end
 
 function Smoke.draw()
 	-- draw smoke trail
-	for key, particle in ipairs(Smoke.particles) do
+	for key, _ in ipairs(Smoke.particles) do
         local particle = Smoke.particles[key]
```
Comment (MadByteDE) [inline source/objects/smoke.lua:70]: Nope, thats my mistake. Just remove `local particle = Smoke.particles[key]` instead of changing particle to _. this line was used because I previously didn't use ipairs to iterate but instead a basic for loop like 
```lua
for key=1, #Smoke.particles do
local particle = Smoke.particles[key]
end
```
Which can be done, but wasn't needed in that case and I seemingly forgot to remove the local definition.

File: source/scripts/enum.lua [hunk 6]
```diff
@@ -14,6 +14,6 @@ return {
     basetypeBuilding1 = 7,
     basetypeBuilding2 = 8,
     baseMaxFuel = 15,
-	
-	
+
+
 }
```
Comment (MadByteDE) [inline source/scripts/enum.lua:19]: Why did you add two new lines here? Personally I prefer writing stuff clean and compact as much as possible.

**Review by MadByteDE — CHANGES_REQUESTED** [review]
Some changes need to be made, see comments.

File: source/main.lua (hunk 4 above)
Comment (togfoxy) [reply source/main.lua:195]: Will it free up memory? If not we can take it out.

Comment (MadByteDE) [reply source/main.lua:195]: Like 2kb maybe? I don't think setting anything to nil should be needed. For objects we just need to make sure they get removed from tables and big assets should be freed when they are not needed, but nothing we currently have is that  big.

Comment (togfoxy) [reply source/main.lua:195]: Okay. Maybe Phil can delete that one line for me.

File: source/main.lua (hunk 3 above)
Comment (Philbywhizz) [reply source/main.lua:249]: I removed it because it was shadowing the local variable for main in line 136 and it made do sense redefining the same thing.

File: source/objects/lander.lua (hunk 2 above)
Comment (Philbywhizz) [reply source/objects/lander.lua]: That was my mistake, I've now reverted it back. :)

File: source/objects/lander.lua (hunk 1 above)
Comment (Philbywhizz) [reply source/objects/lander.lua]: I've reverted these 2 lines. They were shadowing, not unused.

File: source/scripts/enum.lua (hunk 6 above)
Comment (Philbywhizz) [reply source/scripts/enum.lua:19]: I didn't add new lines, I removed the whitespace (tabs)

File: source/main.lua (hunk 4 above)
Comment (Philbywhizz) [reply source/main.lua]: deleted

File: source/objects/smoke.lua (hunk 5 above)
Comment (Philbywhizz) [reply source/objects/smoke.lua]: Fixed.

**Comment by Philbywhizz** [comment]
Changes fixed up now.

**Comment by togfoxy** [comment]
So we are intentionally making
strCurrentScreen

a global? I'm just making sure the internet is not getting in the way of being on the same page?

**Comment by Philbywhizz** [comment]
It's not a global, its a local variable to main.lua

**Comment by togfoxy** [comment]
Right. I see now. I didn't notice. Did I make that local to the module? I forget. Lolz.

**Comment by togfoxy** [comment]
I didn't play test this but the changes are minimal.
This PR is older than the other PR so I'll merge it now. Thanks!
